core_protocol_working_group/meetings/meeting_template.md

```diff
@@ -0,0 +1,23 @@
+
+# Meeting 1
+
+## Date: Feb 14th, 2024, Wednesday, 9-10 AM PST
+
+## How to add an agenda topic?
+Any attendee can add an agenda topic by creating a pull request on this file and adding more topics in the agenda below.
```

Comment:
Instead of this cluttering up every individual meeting file, I feel this should go into the  main readme: https://github.com/onflow/Flow-Working-Groups/blob/main/README.md

Reply:
yup - in that case there will be no `2024-00-00_cp-wp_meeting-template.md`